models/things.js

```diff
@@ -145,4 +145,8 @@ var Things = {
    }
  };
 
+AdapterManager.on('thing-added', function(thing) {
+  Things.handleNewThing(thing);
+});
```

Comment:
See my comments in #99 regarding pairing multiple devices at a time. In principle this makes sense, but we need to think about the potential implications for other adapters.